taggit/models.py

```diff
@@ -26,6 +50,9 @@ class TagBase(models.Model):
         allow_unicode=True,
     )
 
+    natural_key_fields = ["name"]
+    objects = NaturalKeyManager(natural_key_fields)
```

Comment:
I think it should be possible for `NaturalKeyManager` to look up the keys through the `Model` it is managing (probably `self.model` in the context of a manager?)

If we did that then we could just have:
```
natural_key_fields = ["name"]
objects = NaturalKeyManager()
```

Reply:
Yep makes sense, let me clean this up